source/command-monitoring/tests/unified/find.yml

```diff
@@ -63,31 +63,29 @@ tests:
         object: *collection
         arguments:
           filter: { _id: { $gt: 1 } }
-          sort: { _id: 1 }
+          sort: { x: -1 }
```

Comment:
are the sort and projection changes related to the original issue you ran into, or is it just to get some additional coverage?

Reply:
Unrelated.  There was no projection coverage (though really all of this should be in CRUD spec tests)